src/FordhamBank/Aggregates/Transaction.java

```diff
@@ -2,28 +2,42 @@
 
 import java.util.Date;
 
+import FordhamBank.Enums.TransactionType;
+
 public class Transaction extends Aggregate {
     private Date TransactionDate;
     private String TransactionName;
     private double Amount;
+    private double CurrentBalance;
+    private TransactionType TransactionType;
 
-    public Transaction(Date transactionDate, String transactionName, double amount) {
+    public Transaction(Date transactionDate, String transactionName, double amount, double balance, TransactionType type) {
         super();
 
         TransactionDate = transactionDate;
         TransactionName = transactionName;
         Amount = amount;
+        CurrentBalance = balance;
+        TransactionType = type;
     }
 
-    public Date GetTransactionDate() {
+    public Date getTransactionDate() {
```

Comment:
Use capitals like this `GetTransactionDate` for public methods just for the sake of consistency cause thats how all other classes are

Reply:
changed because of the tableview requirements